docs/operations/aws-cost-optimization/elasticache-migration-guide.md

```
@@ -0,0 +1,218 @@
+```bash
+# Check your application health endpoint
+curl https://YOUR-API-URL/health | jq '.redis'
+```
+
+Expected output:
+```json
+{
+  "connected": true,
+  "latencyMs": <number>
+}
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)